splice_si_api/src/main/java/com/splicemachine/si/impl/server/SITransactor.java

```diff
@@ -261,7 +261,8 @@ private void releaseLocksForKvBatch(Pair<KVPair, Lock>[] locks){
         }
     }
 
-    private IntObjectHashMap<DataPut> checkConflictsForKvBatch(Partition table,
+    private IntObjectHashMap<DataPut> checkConflictsForKvBatch(TxnView txn,
```

Comment:
It looks like this new parameter is not used, is it?

Reply:
removed